js/huobi.js

```diff
@@ -3159,14 +3153,7 @@ module.exports = class huobi extends Exchange {
             }
         } else if (linear) {
             const first = this.safeValue (data, 0, {});
-            if (cross) {
-                const account = this.account ();
-                account['free'] = this.safeString (first, 'margin_balance', 'margin_available');
-                account['used'] = this.safeString (first, 'margin_frozen');
-                const currencyId = this.safeString2 (first, 'margin_asset', 'symbol');
-                const code = this.safeCurrencyCode (currencyId);
-                result[code] = account;
-            } else if (isolated) {
```

Comment:
just moved this `if (cross)` block in `else` clause below `if (isolated) ...`, because in all above part of the method, the logic is like that `if (isolated) else...` where `cross` is always meant in `else`.